testing/CMakeLists.txt

```diff
@@ -96,6 +96,41 @@ target_include_directories(xfpga-static PUBLIC
   PRIVATE ${OPAE_SDK_SOURCE}/libopae/plugins/xfpga
   PRIVATE ${OPAE_SDK_SOURCE}/libopae)
 
+add_library(opae-c-ase-static STATIC
```

Comment:
Do you plan to continue calling the ase library opae-c-ase?  I think this made sense when ASE mirrored the opae-c API; but now that it's going to be a plugin, this naming may cause some confusion.  Maybe libase.so would make more sense now?

Reply:
Agree and will rename the ASE library.